ckan/templates/base.html

```diff
@@ -67,6 +67,9 @@
     #}
     {%- block styles %}
       {% resource g.main_css[6:] %}
+      {% if h.lang() in g.rtl_languages %}
```

Comment:
Can you use a `h.is_rtl_language()` helper function that checks the current lang and the config options needed? We are trying to limit the stuff that goes into the globals.